src/functionalTest/java/uk/gov/hmcts/reform/wataskmonitor/MonitorTaskJobControllerForInitiationJobTest.java

```diff
@@ -75,6 +75,7 @@ public void task_initiation_job_should_initiate_task_and_taskState_should_be_unc
             common.getTaskVariables(caseworkerCredentials.getHeaders(), taskVariables.getTaskId());
 
         assertEquals(taskVariables.getCaseId(), camundaVariableMap.get("caseId").getValue());
+        assertEquals("reviewTheAppeal", camundaVariableMap.get("taskId").getValue());
```

Comment:
this isn't working in local, expected value is 'processApplication', could you please check it?